opentelemetry-sdk/tests/metrics/test_aggregation.py

```diff
@@ -308,3 +312,281 @@ def test_collect(self):
+            _convert_aggregation_temporality(
+                Sum(
+                    start_time_unix_nano=1,
+                    time_unix_nano=2,
+                    value=3,
+                    aggregation_temporality=AggregationTemporality.DELTA,
+                    is_monotonic=False,
+                ),
+                Sum(
+                    start_time_unix_nano=4,
+                    time_unix_nano=5,
+                    value=6,
+                    aggregation_temporality=AggregationTemporality.DELTA,
+                    is_monotonic=False,
+                ),
+                AggregationTemporality.CUMULATIVE,
+            ),
+            Sum(
+                start_time_unix_nano=1,
+                time_unix_nano=5,
+                value=9,
+                aggregation_temporality=AggregationTemporality.CUMULATIVE,
+                is_monotonic=False,
+            ),
+        )
+
+        self.assertEqual(
+            _convert_aggregation_temporality(
+                Sum(
+                    start_time_unix_nano=1,
+                    time_unix_nano=2,
+                    value=3,
+                    aggregation_temporality=AggregationTemporality.DELTA,
+                    is_monotonic=True,
+                ),
+                Sum(
+                    start_time_unix_nano=4,
+                    time_unix_nano=5,
+                    value=6,
+                    aggregation_temporality=AggregationTemporality.DELTA,
+                    is_monotonic=False,
+                ),
+                AggregationTemporality.CUMULATIVE,
+            ),
+            Sum(
+                start_time_unix_nano=1,
+                time_unix_nano=5,
+                value=9,
+                aggregation_temporality=AggregationTemporality.CUMULATIVE,
+                is_monotonic=False,
+            ),
+        )
+
+        self.assertEqual(
+            _convert_aggregation_temporality(
+                Sum(
+                    start_time_unix_nano=1,
+                    time_unix_nano=2,
+                    value=3,
+                    aggregation_temporality=AggregationTemporality.DELTA,
+                    is_monotonic=True,
+                ),
+                Sum(
+                    start_time_unix_nano=4,
+                    time_unix_nano=5,
+                    value=6,
+                    aggregation_temporality=AggregationTemporality.DELTA,
+                    is_monotonic=True,
+                ),
+                AggregationTemporality.CUMULATIVE,
+            ),
+            Sum(
+                start_time_unix_nano=1,
+                time_unix_nano=5,
+                value=9,
+                aggregation_temporality=AggregationTemporality.CUMULATIVE,
+                is_monotonic=True,
+            ),
+        )
+
+    def test_current_point_gauge(self):
+
+        current_point = (Gauge(0, 0),)
+        self.assertEqual(
+            _convert_aggregation_temporality(
+                Sum(
```

Comment:
The previous point would be a gauge also

Reply:
fixed